tools/api-markdown-documenter/src/api-item-transforms/Utilities.ts

```diff
@@ -123,6 +127,12 @@ function resolveSymbolicLink(
 
 		return undefined;
 	}
+	const resolvedApiItem = resolvedReference.resolvedApiItem;
+
+	// Return undefined if the resolved API item should be excluded based on release tags
+	if (!shouldItemBeIncluded(resolvedApiItem, config)) {
+		return undefined;
```

Comment:
Nit: It would probably be nice to add a `logger.verbose` call here noting that the link points to something that won't be included in the generated documentation suite. We do a warning above for the case where the link can't be resolved. The former is more likely a bug, whereas this is more likely to be legitimate. So a `verbose` log event seems appropriate.